packages/mgt-chat/src/statefulClient/GraphNotificationClient.ts

```diff
@@ -91,7 +91,9 @@ export class GraphNotificationClient {
     if (this.cleanupTimeout) this.timer.clearTimeout(this.cleanupTimeout);
     if (this.renewalTimeout) this.timer.clearTimeout(this.renewalTimeout);
     this.timer.close();
-    await this.unsubscribeFromChatNotifications(this.chatId, this.sessionId);
+    if (this.chatId && this.sessionId) {
```

Comment:
when would chatId and sessionId be undefined? it appears to be set as empty string during init

Reply:
This is a "falsy" check. It will be true for null, undefined, or empty.